src/keys-api/keys-api.service.ts

```diff
@@ -54,4 +60,33 @@ export class KeysApiService {
       throw Error('Keys API not synced, please wait');
     return result;
   }
+
+  /**
+   *
+   * @param The /v1/keys/find KAPI endpoint returns a key along with its duplicates
+   * @returns
+   */
+  public async getKeysWithDuplicates(pubkeys: string[]) {
```

Comment:
It doesn't return duplicates. It returns all keys entities with provided public keys

Reply:
it return keys with duplicates

Reply:
Maybe findKeyEntities or Entries

Reply:
fixed